src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Views/StudyResultView.sql

```diff
@@ -0,0 +1,32 @@
+﻿/*************************************************************
+    Study View to be used for getting computed column response
+**************************************************************/
+IF NOT EXISTS 
+(
+    SELECT *
+    FROM    sys.views
+    WHERE   Name = 'StudyResultView'
+)
+BEGIN
+    EXEC('CREATE VIEW dbo.StudyResultView
+    WITH SCHEMABINDING
+    AS
+    SELECT  st.StudyInstanceUid,
+            st.PatientId,
+            st.PatientName,
+            st.ReferringPhysicianName,
+            st.StudyDate,
+            st.StudyDescription,
+            st.AccessionNumber,
+            st.PatientBirthDate,
+            (SELECT STRING_AGG(Modality, '','')
```

Comment:
Since Modality can be null. Should we check and do coalesce?

Reply:
Good point, handled it in the store layer